cinema/views.py

```diff
@@ -21,24 +24,49 @@
 )
 
 
-class GenreViewSet(viewsets.ModelViewSet):
+class GenreViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
+):
     queryset = Genre.objects.all()
     serializer_class = GenreSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
 
 
-class ActorViewSet(viewsets.ModelViewSet):
+class ActorViewSet(
+    viewsets.GenericViewSet,
+    mixins.ListModelMixin,
+    mixins.CreateModelMixin
+):
     queryset = Actor.objects.all()
     serializer_class = ActorSerializer
+    authentication_classes = (TokenAuthentication,)
+    permission_classes = (IsAdminOrIfAuthenticatedReadOnly,)
```

Comment:
You may use DEFAULT_PERMISSION_CLASSES & DEFAULT_AUTHENTICATION_CLASSES settings.
![image](https://github.com/mate-academy/py-token-authentication/assets/93856423/f9d5fa49-081e-41e0-993d-4da3974a8343)

Reply:
If you're using the default permission and authentication classes in your view, you can simplify your code by omitting these classes from the view definition. Fix everywhere, please
```suggestion
```